cmd/sample/tag-created-vms/README.md

```diff
@@ -0,0 +1,132 @@
+# Tag Created VMs
```

Comment:
let's move this to:
samples/tag-created-vms/README.md

Reply:
done